modules/learn/pages/services-and-indexes/indexes/index-replication.adoc

```diff
@@ -90,3 +102,30 @@ If index-maintenance is running behind, the query waits for it to catch up.
 
 For N1QL, the default consistency is `not_bounded`.
 // end::scan_consistency[]
+
+[[index-snapshots]]
+== Index Snapshots
+
+A {database-change-protocol}[snapshot] of the index is maintained on disk, to permit rapid recovery if node-failures are experienced.
+In cases where recovery requires an Index-Service node to be restarted, the node’s indexes are rebuilt from the snapshots retained on disk.
+
+You can change the number of committed rollback points for memory-optimized index storage via the {index-storage-mode}[UI].
+You can change more index snapshot settings via the {index-storage-settings-via-cli}[CLI] or the {index-settings-via-rest}[REST API].
+
+[[index-rollback]]
+=== Index Rollback
+
+When a data node {automatic-failover}[fails over automatically], a replica data node is promoted to active.
```

Comment:
This applies to both auto-failover or a user initiated failover.

Reply:
Done